src/starcounter-debug-aid.html

```diff
@@ -173,6 +81,11 @@
                 //this.$.errorDialog.reset();
             },
             openDialog: function () {
+                if (!Polymer.isInstance(this.$.ui)) {
+                    Polymer.Base.importHref("src/starcounter-debug-aid-ui.html", () => {
```

Comment:
I think you need to investigate deeper. Maybe this is a bug in Polymer or Polyserve. Can you explain if Polymer makes any transformations on URLs provided to `importHref`? I have looked in the Polymer issue board and cannot find issues reported to it, but maybe there should be.